### PR TITLE
Parser implementation for unassign

### PR DESCRIPTION
**Pull request by Jasperabez**
Copy the implementation of assign command over to unassign, create dummy functions (TODO) for those needing to access the new associations model.

Tests currently being worked on

Resovles: #85

### CODE REVIEW

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -43,7 +43,8 @@ public ModelManager() {
         this(new AddressBook(), new UserPrefs());
     }
 
-    //=========== UserPrefs ==================================================================================
+    // =========== UserPrefs
+    // ==================================================================================
```
Comment (C5hives) [inline src/main/java/seedu/address/model/ModelManager.java:47]: Need to revert this change by your auto formatter.

File: src/main/java/seedu/address/model/AddressBook.java [hunk 2]
```diff
@@ -22,18 +22,22 @@ public class AddressBook implements ReadOnlyAddressBook {
     private final UniqueEventList events;
 
     /*
-     * The 'unusual' code block below is a non-static initialization block, sometimes used to avoid duplication
-     * between constructors. See https://docs.oracle.com/javase/tutorial/java/javaOO/initial.html
+     * The 'unusual' code block below is a non-static initialization block,
+     * sometimes used to avoid duplication
+     * between constructors. See
+     * https://docs.oracle.com/javase/tutorial/java/javaOO/initial.html
```
Comment (C5hives) [inline src/main/java/seedu/address/model/AddressBook.java:28]: You might want to change how your auto formatter runs.

You and @hanscau will end up changing the javadocs back and forth for every created PR.

**Review by C5hives — CHANGES_REQUESTED** [review]
Looks good except for some formatting changes that need to be addressed

File: src/main/java/seedu/address/model/AddressBook.java [hunk 3]
```diff
@@ -108,10 +115,19 @@ public void removeVendor(Vendor key) {
         vendors.remove(key);
     }
 
+    boolean isVendorAssignedToEvent(Vendor vendor, Event event) {
+        return false; // TODO implement this
+    }
```
Comment (kevin-pek) [inline src/main/java/seedu/address/model/AddressBook.java:120]: I have just merged the PR for the assign command, so this method can now be removed!

File: src/main/java/seedu/address/model/Model.java [hunk 4]
```diff
@@ -93,6 +93,10 @@ public interface Model {
      */
     void updateFilteredVendorList(Predicate<Vendor> predicate);
 
+    boolean isVendorAssignedToEvent(Vendor vendor, Event event);
```
Comment (kevin-pek) [inline src/main/java/seedu/address/model/Model.java:96]: Ditto.

**Review by kevin-pek — CHANGES_REQUESTED** [review]

**Review by kevin-pek — APPROVED** [review]

**Review by C5hives — APPROVED** [review]
Looks good!

**Pull request merged**